Cubical/Core/Prelude.agda

```diff
@@ -45,6 +45,11 @@ cong : ∀ {B : A → Set ℓ'} (f : (a : A) → B a) (p : x ≡ y)
        → PathP (λ i → B (p i)) (f x) (f y)
 cong f p = λ i → f (p i)
 
+-- Less polymorphic version of `cong`, to avoid some unresolved metas
+cong′ : ∀ {B : Set ℓ'} (f : A → B) {x y : A} (p : x ≡ y)
```

Comment:
This should go into a suitable file in `Foundations` and not the core. I'm not sure which file though... Where does `cong` go in the standard library?

Reply:
Relation.Binary.PropositionalEquality

Reply:
Ok, so maybe we should add a `Relation.Binary.Path` file.

Reply:
I would rather go for `Cubical.Equality` or `Cubical.Foundations.Equality`.

It's always been odd to go into `Relation.Binary.*` for equality.

Reply:
That is also fine with me. But can we please put it in `Cubical.Foundations.Paths` (or `.Path`)?

I think a lot of confusion that I've encountered among mathematicians when trying to explain what we are doing comes from the fact that we are using the word "equality" for paths. The problem with this is that "equality" in standard mathematical practice means (definitionally) proof-irrelevant equality. So the idea that univalence says "equivalence is equality" sounds really quite absurd. However if we make it clear that we are working with **proof-relevant** equality then it suddenly makes a lot more sense as equality is structure and not property. I like the idea of calling this notion "paths" following what Voevodsky did in his Foundations library.

Reply:
That's fine for me

Reply:
Do we move the more polymorphic cong as well? What about sym, compPath?

Reply:
Created `C.F.Path` and moved `cong'`.

Reply:
Let's not do it in this PR, but it might make sense to shrink the Core a bit as long as the HoTT-UF file still compiles